HW1/P6/P6.py

```diff
@@ -0,0 +1,73 @@
+from pyspark import SparkContext
+import random
+
+# define a function to account for increased chance of appearing if seen multiple times
+def thirdWord(tup):
+    thirdWords = map.value[tup]
+    words = []
+    for (x,c) in thirdWords:
+        for i in xrange(c):
+            words.append(x)
+    return random.choice(words)
```

Comment:
Can probably use numpy's random.choice, to clarify the code and avoid lists with redundant entries.